### ISSUE TITLE
remove double "backspace" press 

### ISSUE THREAD

**Issue by ddtch** [pkovzz/ngx-otp-input#9]
**Describe the bug**
So it is not an actual bug, but when I press backspace once on filled input it removes the value but doesn't go to the previous cell. 

**To Reproduce**
1. Go to https://ngx-otp-input.vercel.app/
2. Click in the first cell and start typing
3. remove last cell value
4. See that focus still in same (empty cell)

**Expected behavior**
So in my logic and best practices is next - When I deleted the value of the cell, it should focus on the previous cell, and if I type a new value should be checked if the cell does not empty populate new value to the next empty cell otherwise fill new value in current

**Desktop (please complete the following information):**
 - Browser chrome
 - Version 95

**Smartphone iPhone iOS 15:**
 - Browser safari

**Additional context**

**Comment by pkovzz** [comment]
Hi @ddtch!

Why should it focus on the previous cell? The current behaviour is nearly identical to your description. Because if your previous cell has a value, the input would jump forward anyway, right? If I misunderstood your request, let's discuss it! :)

**Comment by ddtch** [comment]
It should BCS in case I want to delete 4 symbols I will press backspace 8 times. this is weird UX.
But if I delete the last value and go to previous cell, and will type something under the hood should be checked like 'if current cell has value -> populate value in next empty cell'

Just try to do this in any bank apps or so where we have classic otp inputs

**Comment by pkovzz** [comment]
I could add a new option for the desired behaviour. The current one remains the default.

**Comment by pkovzz** [comment]
I published a new version of the lib, see the commit here: https://github.com/pkovzz/ngx-otp-input/commit/8cec872bbb7171fdacd0fdf29f42960c7212d02f

In the end, I created a hybrid solution. By default, the deletion will work as we discussed, but add new values remains the same as was. So as I mentioned, this new approach is the default behaviour now, but I also added a new optional property called `behavior`, which can toggle between the old and new behaviours.
